### DISCUSSION TITLE
XXEN_UTIL

### DISCUSSION THREAD

**Discussion by amodeoms** [Enginatics/Oracle-EBS-SQL#2] [Q&A] [answered]
Our vendor DBA support tried to run the script against our DEV environment and received a compilation error ... is the current script Oracle version specific?
![xxen_util_error](https://github.com/Enginatics/Oracle-EBS-SQL/assets/146244430/7f1c70ea-08cf-4aa3-b2c9-9dc035f76eb7)

**Comment by artem-nesterko** [comment, marked as answer]
Hello @amodeoms,

xxen_util is not specific to Oracle versions, but you would also need the package specification (.pks file). It was not included in this repository, but you could obtain the latest version from the Blitz Report installation archive (https://www.enginatics.com/download/#_download). The latest xxen_util.pkb and xxen_util.pks files can be found in the /sql folder.

Furthermore, I've added the latest xxen_util.pkb and xxen_util.pks files to this repository. You can already locate them in the same location as xxen_util.sql (root folder).
![image](https://github.com/Enginatics/Oracle-EBS-SQL/assets/1782544/390f9adf-78ba-4b31-afb0-5568ff00b430)

You would need just to run xxen_util.pks and xxen_util.pkb against your DEV environment.

**Comment by amodeoms** [comment]
Thank you. I've asked our support vendors to apply those to our DEV environment.
